cli/azd/cmd/provision.go

```diff
@@ -48,8 +55,13 @@ func (i *provisionFlags) setCommon(envFlag *envFlag) {
 	i.envFlag = envFlag
 }
 
-func newProvisionFlags(cmd *cobra.Command, global *internal.GlobalCommandOptions) *provisionFlags {
-	flags := &provisionFlags{}
+func newProvisionFlags(
+	cmd *cobra.Command,
+	global *internal.GlobalCommandOptions,
+	alphaFeatureManager *alpha.FeatureManager) *provisionFlags {
+	flags := &provisionFlags{
+		alphaFeatureManager: alphaFeatureManager,
+	}
```

Comment:
Do we still have both `provision` and `infra create`? I thought we deprecated `infra create` ??